src/org/omegat/gui/preferences/view/EditingBehaviorController.java

```diff
@@ -72,10 +72,10 @@ private void initGui() {
 
     @Override
     protected void initFromPrefs() {
-        panel.defaultRadio.setSelected(!Preferences.isPreference(Preferences.DONT_INSERT_SOURCE_TEXT));
-        panel.leaveEmptyRadio.setSelected(Preferences.isPreference(Preferences.DONT_INSERT_SOURCE_TEXT));
+        panel.defaultRadio.setSelected(Preferences.isPreference(Preferences.DONT_INSERT_SOURCE_TEXT));
+        panel.leaveEmptyRadio.setSelected(!Preferences.isPreference(Preferences.DONT_INSERT_SOURCE_TEXT));
```

Comment:
```suggestion
        panel.defaultRadio.setSelected(Preferences.isPreferenceDefault(Preferences.DONT_INSERT_SOURCE_TEXT, false));
        panel.leaveEmptyRadio.setSelected(!Preferences.isPreferenceDefault(Preferences.DONT_INSERT_SOURCE_TEXT, true));
```

Reply:
Maybe the default true/false values can be refactored as a `Preferences.DONT_INSERT_SOURCE_TEXT_DEFAULT` field, but maybe it's overkill